sleap/gui/commands.py

```
@@ -3505,13 +3516,230 @@
+        for labeled_frame in pruned_labels.labeled_frames:
+            labeled_frame.frame_idx -= frame_offset
+
+        # Initialize VideoWriter
+        height, width = video.height, video.width
+        fps = params["fps"]
+        writer = VideoWriter.safe_builder(params["filename"], height, width, fps)
+
+        # Conditionally show progress bar
+        show_progress = os.getenv("PYTEST_RUNNING") != "1"
+        if show_progress:
+            app = QtWidgets.QApplication.instance() or QtWidgets.QApplication([])
```

Ruff (0.8.2)

Local variable `app` is assigned to but never used

Remove assignment to unused variable `app`

(F841)